### ISSUE TITLE
Disable the RuboCop ElseAlignment or ConditionalAssignment cop by default

### ISSUE THREAD

**Issue by oliverklee** [brigade/haml-lint#114]
In one of our HAML templates. we have the following code:

```haml
- if user.full_name?
  - name_link_text = user.full_name
- else
  - name_link_text = t('views.admin.noName')
```

Running haml_lint with the default configuration creates this warning:
```
app/views/admin/users/index.html.haml:46 [W] RuboCop: Use the return of the conditional for variable assignment and comparison.
```

(This is the RuboCop ConditionalAssignment cop AFAICT.)

Changing the code as recommended:

```haml
- name_link_text = if user.full_name?
  - user.full_name
- else
  - t('views.admin.noName')
```

Now haml_lint (i.e., RuboCop) complains about the alignment:

```
app/views/admin/users/index.html.haml:48 [W] RuboCop: Align `else` with `if`.
```

This probably is the ElseAlignment cop.

Changing the indentation causes haml_lint to complain about the indentation:

```
app/views/admin/users/index.html.haml:47 [E] Inconsistent indentation: 37 spaces used for indentation, but the rest of the document was indented using 2 spaces.
```

I propose disabling the ElseAlignment or ConditionalAssignment cop for haml_lint (preferably the ElseAlignment cop).
